File: sieve.py
import math

target = 1000
integers_to_target = []

for i in range(2, target):
	integers_to_target.append(i)

for i in range(2, math.ceil(math.sqrt(target))):
	for j in range(2, math.floor(target/i)):
		integers_to_target[(i * j) - 2] = 0

for i in range(0, (target - 2)):
	if integers_to_target[i] != 0:
		print(integers_to_target[i])

File: sieve.c
#include <stdio.h>
#include <math.h>

int main(){
	const int target = 1000;
	int integers_to_target[target - 2];

	for (int i = 2; i <= target; i++) {
		integers_to_target[i-2] = i;
	}

	for (int i = 2; i < sqrt(target); i++) {
		for (int j = 2; j < target/i; j++) {
			integers_to_target[(i * j) - 2] = 0;
		}
	}

	for (int i = 0; i <= (target-2); i++) {
		if (integers_to_target[i] != 0) {
			printf("%d", integers_to_target[i]);
			printf("\n");
		} else {
			continue;
		}
	}

	return 0;

}
